buildSrc/src/main/kotlin/kotlin-common.gradle.kts

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
Can we please not call anything with "common"? To me this file looks more like "configure-kotlin", "kotlin-settings", or something similar.